### ISSUE TITLE
get_feature_importance(fstr_type='ShapValues') causes unknown pointer encoding

### ISSUE THREAD

**Issue by idavydov** [catboost/catboost#384]
Problem: get_feature_importance (ShapValues) throws an error after training.
```
libunwind: getEncodedP /home/travis/build/catboost/catboost/contrib/libs/libunwind_master/src/AddressSpace.hpp:269 - unknown pointer encoding
```
Here's a minimal example:
```
from catboost import CatBoostRegressor, Pool
import pandas as pd


df = pd.read_csv('data.txt')

cat_features = [i for i, (col, ser) in enumerate(df.iteritems()) if ser.dtype == object]


pool = Pool(data=df.iloc[:, :-1].values, label=df.iloc[:, -1].values, cat_features=cat_features)

model=CatBoostRegressor(iterations=100, verbose=True)
model.fit(pool)

shap_values = model.get_feature_importance(data=pool, fstr_type='ShapValues')

print(shap_values)
```

[data.txt](https://github.com/catboost/catboost/files/2099303/data.txt) (anonymized and randomized)

The error did not reproduce on a smaller number of iterations or smaller number of samples.

catboost version: 8.1.1, python3.6
Operating System: GNU/Linux x86_64
CPU: Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz

**Comment by annaveronika** [comment]
Do you have categorical features in your dataset? If yes, then how many?
Could you provide your dataset or reproduce the error with some dataset that you can generate and provide code for generation? So that we can debug the error.

**Comment by annaveronika** [comment]
Could you also try to rebuild the package from source? It looks like a problem with how package was built.

**Comment by annaveronika** [comment]
Normally you would get exception with text "Categorical feature combinations are not supported jet for Shap values" or some similar text.
To overcome this exception you can use ctr_complexity=1 - then you will not have combinations.

Shap values for combinations will be added soon - they are currently on review in our internal repo. They will be on github tomorrow or after tomorrow. Then you will be able to use them by building from source or wait for our next release on github.

**Comment by idavydov** [comment]
@annaveronika, thanks. I can confirm that `max_ctr_complexity=1` solves the issue, so most probably it's feature combinations as you suggested.

Since you have #314 for feature combinations, this one can be closed.

**Comment by idavydov** [comment]
I tried compiling catboost from source (624acae, latest master at the moment) following the instructions you provided, w/o CUDA support.

Unfortunately I still get this error on the example from the issue:
```
libunwind: getEncodedP /<my home>/src/catboost/contrib/libs/libunwind_master/src/AddressSpace.hpp:269 - unknown pointer encoding
Aborted
```

I guess it makes sense to reopen it, since there should be an error message.

P.S. I just noticed that the python package [has a version '0.8.1'](https://github.com/catboost/catboost/blob/master/catboost/python-package/catboost/version.py) in the latest master, however the latest release is 8.1.1. Is it supposed to be like this?
